controller/app.controller.js

```diff
@@ -0,0 +1,17 @@
+
+
+const {selectTopics} = require('../model/app.model')
+
+
+
+exports.getTopics = (req,res,err)=> {
```

Comment:
The naming here is misleading - we can pass req, res, and next to this function but not err. Our catch block will give us access to an error if there is one